app/src/main/java/com/amaze/filemanager/activities/MainActivity.java

```diff
@@ -856,6 +851,15 @@ public void invalidatePasteButton(MenuItem paste) {
 
   public void exit() {
     if (backPressedToExitOnce) {
+
+      //            if(ExplorerDatabase.getInstance() != null &&
```

Comment:
Can remove commented code if not needed? BTW not closing the connection on exit?